lms/static/scripts/frontend_apps/config.js

```diff
@@ -71,6 +71,7 @@ import { createContext } from 'preact';
  * @typedef FilePickerConfig
  * @prop {string} formAction
  * @prop {Record<string,string>} formFields
+ * @prop {APICallInfo} deeplinkingAPI
```

Comment:
It looks like this is optional in the backend? If so this should be `[deeplinkingAPI]`, which is equivalent to setting the type to `APICalInfo|undefined`. This will result in an error if you try to use it without checking if it is set.